### ISSUE TITLE
Upload JSON button creates a unstyled close button

### ISSUE THREAD

**Issue by BoyWithSilverWings** [BoyWithSilverWings/json-formatter#6]
Solution: 
1. Remove default appearance of the button with just the icon
2. style on keyboard focus
3. should work on Enter

**Comment by JGriffCS** [comment]
Hey @BoyWithSilverWings, I'd like to take a look at this, but I want to clarify your expectations:

1) By this do you mean removing the "button" properties and only displaying an "X" as though it were an icon?
2) What do you mean by "style on keyboard focus"? Do you mean that some type of "focus" styles should be applied when the element is focused via tab/keyboard (I guess it isn't at the moment?). If so, what did you have in mind?
3) When you say it "should work on enter" do you mean that when the element is focused that the enter key should trigger the click event? If so it seems this is already working, although you have to guess as to whether or not the button is focused due to what I assume you're describing in issue 2.

Thanks in advance!

**Comment by JGriffCS** [comment]
Perhaps something like this?

Default:
![Clear Button Unfocused](https://user-images.githubusercontent.com/7431677/66768730-404e2100-eeee-11e9-8a51-302060dc430f.PNG)

Focused:
![Clear Button](https://user-images.githubusercontent.com/7431677/66768648-0b41ce80-eeee-11e9-86f3-d7592033bac7.PNG)

**Comment by BoyWithSilverWings** [comment]
@JGriffCS  Can you move the button to the right of 1 file selected, looks like it makes sense there. add some gap for good measure

**Comment by JGriffCS** [comment]
Not a problem. 5px looks right to me:

![Inline Clear](https://user-images.githubusercontent.com/7431677/66769283-adae8180-eeef-11e9-8514-a238b66b3d23.PNG)
